ui/static/js/fortio_chart.js

```diff
@@ -189,6 +189,7 @@ function fortioResultToJsChartData (res) {
 
 function showChart (data) {
   toggleVisibility()
+  deleteSingleChart()
```

Comment:
This line unfortunately removes the cute transition animation when switching between single results. To keep them, there needs to be some better management of the 2 "overlayed" datasets in the global chart variable.

Reply:
thanks for pointing this out ! let's try to keep the "using up/down arrow" to move from result to result working nicely:

add an if ? there is already logic to switch from multi to single - we just now have 3 modes and as long as we stay in either one of the mode the only change is the data ?

Reply:
This line resolves the bug that happens when moving from an overlay chart back to a single chart. Without this line, the data isn't reset and the single chart shows 4 datasets.

It only needs to be called when going from an overlay chart back to a single chart, however, so I added an if check.